tests/unit-tests/test_config.py

```diff
@@ -480,7 +480,8 @@ def test_train_info():
         check_failure(config, "batch_size")
         config._batch_size = 32
         assert config.batch_size == 32
-        assert config.eval_batch_size == 32
```

Comment:
You do not need to set it to 10000.
By default, it will be 10000. You are testing its default value.

Reply:
Gotcha!